### DISCUSSION TITLE
AIS-Catcher and YAAC?

### DISCUSSION THREAD

**Discussion by einsteine77** [jvde-github/AIS-catcher#202] [Q&A]
I've been having a tough time trying to get AIS-Catcher to work with YAAC. YAAC docs say that it can receive Direwolf AIS messages via a plugin. Is there any way to change the format the messages that AIS-Catcher sends to YAAC? YAAC does have a TCP plugin for NEMA but it does not seem to want to talk with AIS-Catcher.  

YAAC is looking for a string that looks like this:

[0.3 08:51:47] AIS>APDW16:{DA!AIVDM,1,1,,A,15N4cJ`000rrjq@H@0i2hbuN0L0U,0*31

When I get to the part in the readme file where is describes formatting, the readme refers to the "full documentation", which I'm having trouble locating...

Any help is greatly appreciated.

Best Regards

Jeremy N2UGS

**Comment by jvde-github** [comment]
I see that format is AIS output of Direwolf: https://github.com/wb2osz/direwolf/blob/master/doc/AIS-Reception.pdf
Does not seem complicated to produce from a standard AIS message with a bit of Python in between.

What I am not clear about how the data is send between Direworld and YAAC. Sorry, I don't have experience with these programs.

**Reply by einsteine77** [reply]
Great! How do I get started?

**Reply by jvde-github** [reply]
If you have these message strings for example on the console, how do you plan to get them into YAAC?
